docs/develop/debugging-cloe.rst

```diff
@@ -5,31 +5,82 @@ Debugging is part of development, and unless you're the perfect developer, you
 will probably want to do this from time to time. This section contains a few
 tips for debugging Cloe components and Cloe as a system.
 
+Conan Profiles
```

Comment:
Suggestion: On the top of the page, add
```
.. note::
Please ensure you :doc:`understand<../usage/using-cloe-launch>`
how `cloe-launch` works before reading this section.
```

Reply:
`testing-cloe`would be a good pre-read as well, I think.

Reply:
done